docs/DeveloperGuide.md

```diff
@@ -19,19 +28,16 @@ title: Developer Guide
   - [Common classes](#common-classes)
 - [**Implementation**](#implementation)
   - [Implemented features:](#implemented-features)
-    - Record features:
-      - [Add record feature](#implemented-add-record-feature)
-      - [List records feature](#implemented-list-records-feature)
-      - [Delete record feature](#implemented-delete-record-feature)
-      - [Edit record feature](#implemented-edit-record-feature)
-      - [Find records feature](#implemented-find-records-feature)
-      - [Clear all records feature](#implemented-clear-all-records-feature)
-    - Appointment features:
-      - [Add appointment feature](#implemented-add-appointment-feature)
-      - [Clear appointment feature](#implemented-clear-appointment-feature)
+    - [Add record feature](#add-record-feature)
```

Comment:
I thought we agreed to leave the record and appointment feature as headers in the TOC, just that it doesn't link to anything?